binnacle/lib/utils/Math.dart

```diff
@@ -7,23 +7,28 @@ double radiansToDegrees(double radians) {
   return radians * 180 / pi;
 }
 
+double degreesToRadians(double degrees) {
```

Comment:
this formula is wrong, Correct way -> degrees * (2*pi )/ 360
or even better -> degrees * (pi) / 180

Reply:
lol